src/main/java/core/BasePage.java

```diff
@@ -66,21 +66,19 @@ public void chooseDataFromDropDownMenu(WebElement webElement, String userData){
 
     }
 
-    public void checkRadioButtonByName(String className, int numberOfRadioButton){
+    public BasePage checkRadioButtonByName(String className, int numberOfRadioButton){
```

Comment:
I don't understand well about "numberOfRadioButton". I don't find the "number..." on the "search page".